.github/workflows/ci.yml

```
@@ -1,30 +1,139 @@
----
-
 name: CI
 
 on:
   pull_request:
-    branches: [devel]
+    branches:
+      - devel
 
   push:
-    branches: [devel]
+    branches:
+      - devel
+
+  schedule:
+    - cron: '00 04 * * *'
+
 
 jobs:
+  sanity:
+    name: ${{ matrix.test.name }}
+    runs-on: ubuntu-20.04
+    container:
+      image: quay.io/ansible/ansible-builder-test-container:1.0.0
+      env:
+        PIP_CACHE_DIR: ${{ runner.temp }}/.cache/pip
+        TOXENV: ${{ matrix.test.tox_env }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          - name: Lint
+            tox_env: linters
+
+          - name: Docs
+            tox_env: docs
```

looking at this: do you really need separate name and tox env? It's usually more than enough to reuse envs as job names:
```suggestion
        tox_env:
        - linters
        - docs
```

---

I could do that, I just like the way it looks having the job name capitalized. GHA doesn't provide an easy mechanism for manipulating the tox environment string into a name that I think looks nice, so that's the reason for separate entries. It's pure cosmetic.

---

FWIW I'm working on a reusable workflow for tox and here's what it'll look like: https://github.com/webknjaz/tox-gha-test/actions/runs/1450588254 — with the exact env names.